README.rst

```diff
@@ -19,6 +19,21 @@ You may obtain a copy of the License at:
 
 https://opensource.org/licenses/MIT
 
+Note
+========
+
+To maintain MOCS table compatibility with i915 Kernel >= 5.1 and older,
+gmmlib supports 2 modes of MOCS programming and they are enabled by compile 
+time flag called 'GMM_KERNEL_5_1_MOCS_TABLE':
+
+-  Usage: FALSE = Default -- Use 3 Gen9 based mocs entries
+   until Kernel 5.1 becomes official or i915.alpha_support for the ICL has been removed from newer kernels
```

Comment:
until Kernel 5.1 becomes official >>>and/<<< or i915.alpha_support for the ICL >>>is<<< removed from newer kernels